src/decode.rs

```diff
@@ -4802,7 +4802,8 @@ unsafe fn rav1d_decode_frame_main(c: &Rav1dContext, f: &mut Rav1dFrameData) -> R
                 rav1d_decode_tile_sbrow(c, &mut t, f).map_err(|()| EINVAL)?;
             }
             if f.frame_hdr().frame_type.is_inter_or_switch() {
-                rav1d_refmvs_save_tmvs(&c.refmvs_dsp, &mut t.rt, 0, f.bw >> 1, t.by >> 1, by_end);
+                let by = t.by; // For borrowck.
+                rav1d_refmvs_save_tmvs(&c.refmvs_dsp, &mut t.rt, 0, f.bw >> 1, by >> 1, by_end);
```

Comment:
I have no idea why I have to do this, since it works fine in the other `rav1d_refmvs_save_tmvs` call when we pass `&mut t.rt` and `t.by > 1`, too.

Reply:
Still have no idea why this happened, but I removed it another way (`&mut t.rt` isn't necessary, only a `&` is needed).